GOT-StudentVersion/GOT-StudentVersion/EpisodeDetailViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  EpisodeDetailViewController.swift
+//  GOT-StudentVersion
+//
+//  Created by Michelle Cueva on 8/13/19.
+//  Copyright © 2019 C4Q . All rights reserved.
+//
+
+import UIKit
+
+class EpisodeDetailViewController: UIViewController {
+    
+    var episode: GOTEpisode!
+    
+    
+    @IBOutlet weak var imageView: UIImageView!
+    
+    @IBOutlet weak var nameLabel: UILabel!
+    
+    @IBOutlet weak var seasonLabel: UILabel!
+    
+    @IBOutlet weak var episodeLabel: UILabel!
+    
+    @IBOutlet weak var runTimeLabel: UILabel!
+    
+    @IBOutlet weak var airDateLabel: UILabel!
+    
+    @IBOutlet weak var descriptionLabel: UITextView!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        getAllVars()
+        descriptionLabel.isEditable = false
+    }
+    
+    func getAllVars() {
```

Comment:
mark this function private! But I liked that you made a separate function for it instead of putting it directly in the viewDidLoad